src/Bynder/Api/Impl/OAuth2/RequestHandler.php

```diff
@@ -37,19 +38,28 @@ public function getAccessToken($code)
 
     protected function sendAuthenticatedRequest($requestMethod, $uri, $options = [])
     {
+        $updatedHeaders = ['headers' => [
+            'User-Agent' => 'bynder-php-sdk/' . $this->configuration->getSdkVersion()
+        ]];
+
         $this->configuration->refreshToken($this->oauthProvider);
+        if (!is_null($options['headers'])) {
+            $updatedHeaders = ['headers' => array_merge(
+                $options['headers'],
+                $updatedHeaders['headers']
+            )];
+        }
```

Comment:
Wouldn't this work if we change the signature to `$options = ['headers': []]`?

(and same below)

```suggestion
        $this->configuration->refreshToken($this->oauthProvider);

        $updatedHeaders = ['headers' => array_merge(
            $options['headers'],
            ['User-Agent' => 'bynder-php-sdk/' . $this->configuration->getSdkVersion()]
        )];
```

Reply:
Yeah makes sense will change the method signature to `$options = ['headers' => []]` 👍